features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha512_alt_sw.c

```diff
@@ -56,8 +50,10 @@
 #endif /* MBEDTLS_SELF_TEST */
```

Comment:
I do not think there is a self test in this file

Reply:
@andresag01 Same as above.